gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -165,6 +165,9 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
             kspExtension.apOptions.forEach {
                 options += SubpluginOption("apoption", "${it.key}=${it.value}")
             }
+            kspExtension.processorName.forEach {
+                options += SubpluginOption("processorName", "$it")
```

Comment:
Is this a task input? Also, consider adding an integration test in which you modify processor names and assert that task is not up-to-date.

Reply:
done, thanks for pointing this out!